### PR TITLE
Pre new k8s cluster update

### PR DESCRIPTION
**Pull request by lepsalex**

### CODE REVIEW

File: helm/elastic-kibana/dev/values.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+nameOverride: "wf-elastic-kibana"
+fullnameOverride: "workflow-elastic-kibana"
+
+ingress:
+  enabled: true
+  annotations: {}
+  path: /
+  hosts:
+    - nf-kibana.light.overture.bio
+  tls: []
```
Comment (Buwujiu) [inline helm/elastic-kibana/dev/values.yaml]: newline

File: helm/elastic-kibana/qa/values.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+elasticsearchHosts: "http://wf-elasticsearch-master:9200"
+
+ingress:
+  enabled: true
+  annotations: {}
+  path: /
+  hosts:
+    - wf-kibana.flow.infra.icgc-argo.org
+  tls: []
```
Comment (Buwujiu) [inline helm/elastic-kibana/qa/values.yaml]: newline

File: helm/elastic-search/dev/values.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+clusterName: "wf-elasticsearch"
+
+ingress:
+  enabled: true
+  annotations: {}
+  path: /
+  hosts:
+    - wf-es.light.overture.bio
+  tls: []
```
Comment (Buwujiu) [inline helm/elastic-search/dev/values.yaml]: newline

File: helm/elastic-search/qa/values.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+clusterName: "wf-elasticsearch"
+
+ingress:
+  enabled: true
+  annotations: {}
+  path: /
+  hosts:
+    - wf-es.flow.infra.icgc-argo.org
+  tls: []
```
Comment (Buwujiu) [inline helm/elastic-search/qa/values.yaml]: newline

File: helm/relay/index/dev/values.yaml [hunk 5]
```diff
@@ -11,7 +11,7 @@ fullnameOverride: "workflow-relay-index"
 appConfig:
   SPRING_PROFILES_ACTIVE: "index"
   SPRING_CLOUD_STREAM_KAFKA_BINDER_BROKERS: "workflow-kafka-cp-kafka-headless:9092"
-  ELASTIC_HOST: "elasticsearch-master-headless"
+  ELASTIC_HOST: "wf-elasticsearch-master-headless"
```
Comment (Buwujiu) [inline helm/relay/index/dev/values.yaml]: why is it not `elasticsearch-master-headless`?

Comment (lepsalex) [reply helm/relay/index/dev/values.yaml]: I renamed this `helm/elastic-search/qa/values.yaml:5`

**Pull request merged**